registerAdBeacon parameters to match the implementation (#326)
WICG/turtledove: Fenced_Frames_Ads_Reporting.md

**Event type and data:** Includes the event type and data associated with an event. When an event type e.g. click matches to the event type registered in registerAdBeacon, the data will be used by the browser as the data sent in the beacon sent to the registered URL.

> **Destination type:** List of values to determine whether this event needs to be reported to buyer, seller or component seller or any combination of those.

Reviewer: Can we clarify how a component seller is specified?  Is it "component seller"?  Perhaps say:
List of values to determine whose registered beacons are reported, can be a combination of `'buyer'`, `'seller'`, or `'component seller'`.

### Parameters

> A map from event type to reporting url, where the event type corresponds to the event type in reportEvent. Event type enables the worklets (for buyer, seller or component seller) to register distinct beacon URLs for different event types. The reporting url is the url to which a beacon will be sent by the browser on receiving events from the fenced frame via reportEvent. The worklet can add the buyerEventId/sellerEventId or any other relevant information as query parameters to this url.

Reviewer 2: Would you say the parameter here is `click` with the value as the reporting URL?

> A map from event type to reporting url, where the event type corresponds to the event type in reportEvent. Event type enables the worklets (for buyer, seller or component seller) to register distinct beacon URLs for different event types. The reporting url is the url to which a beacon will be sent by the browser on receiving events from the fenced frame via reportEvent. The worklet can add the buyerEventId/sellerEventId or any other relevant information as query parameters to this url.

Reviewer 2: ```suggestion
A map from event type to reporting url, where the event type corresponds to the value of reportEvent. The event type enables worklets (for buyer, seller, or component seller) to register distinct beacon URLs for different event types. The reporting url is the location where a beacon is sent once the fenced frame delivers an event via reportEvent.

Worklets can add the buyerEventId, sellerEventId, or any other relevant information as query parameters to this url.
```

### Parameters

> A map from event type to reporting url, where the event type corresponds to the value of reportEvent. The event type enables worklets (for buyer, seller, or component seller) to register distinct beacon URLs for different event types. The reporting url is the location where a beacon is sent once the fenced frame delivers an event via reportEvent.

Reviewer: ```suggestion
A map from event type to reporting URL, where the event type corresponds to the `eventType` value passed to  `reportEvent()`. The event type enables worklets (for buyer, seller, or component seller) to register distinct reporting URLs for different event types. The reporting URL is the location where a beacon is sent once the fenced frame delivers the corresponding event via `reportEvent()`.
```
